tests/CoreTest.php

```diff
@@ -4,19 +4,23 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 use AWS\CRT\CRT;
+use Yoast\PHPUnitPolyfills\TestCases\TestCase;
 
 require_once('common.inc');
 
 // This intentionally does not inherit from CrtTestCase because it needs a clean-room environment
-final class CoreTest extends PHPUnit_Framework_TestCase {
+final class CoreTest extends TestCase
+{
```

Comment:
trivial: We should keep the brace style consistent. Maybe change these to K&R style.